node/cmd/guardiand/adminclient.go

```diff
@@ -549,8 +549,21 @@ func runChainGovernorResetReleaseTimer(cmd *cobra.Command, args []string) {
 	}
 	defer conn.Close()
 
+	// defaults to 1 day if num_days isn't specified
+	var numDays = uint32(1)
```

Comment:
```suggestion
	numDays := uint32(1)
```

shorthand for the same thing